src/test/java/org/owasp/esapi/logging/appender/EventTypeLogSupplierTest.java

```diff
@@ -0,0 +1,26 @@
+package org.owasp.esapi.logging.appender;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+import org.owasp.esapi.Logger;
+import org.owasp.esapi.Logger.EventType;
+
+public class EventTypeLogSupplierTest {
+
+	@Test
+	public void testEventTypeLog() {
+		EventType eventType = Logger.EVENT_UNSPECIFIED;
+		EventTypeLogSupplier supplier = new EventTypeLogSupplier(eventType);
+		
+		assertEquals(eventType.toString(), supplier.get());
+	}
+	
+	@Test
+	public void testNullEventTypeLog() {
```

Comment:
This is new from the PR #530.  Required for the SPI of the Log4J logger to continue functioning.  This will need to be managed when #530 is available on develop

Reply:
Other PR has been closed w/out merge.  This is a non-issue